Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -378,14 +380,16 @@ def url_command(client: Client, url: str) -> Tuple[str, Dict, Union[Dict, list]]
                 'Reliability': client.reliability
             })
     if not raws:
+        if no_matches_url:
+            return f"No matches for URL\S {no_matches_url}", {}, {}
```

Comment:
Why do we need both messages? Can there be a case when "not rows" is true but "no_matches_url" is false?

Reply:
no_matches_url = true only in case the url returns with status_code 404, in all other cases no_matches_url = false but maybe not_rows = true